### ISSUE TITLE
"variable used only once" suppression

### ISSUE THREAD

**Issue by rkday** [SIPp/sipp#24]
SIPped, the Polycom fork of SIPp (https://github.com/SIPp/polycom-sipped) suppresses some "variable used only once" errors where that makes sense. It would be good to get this merged into the main SIPp code.

It looks as though the relevant commits are  ff8ffa2 and 48e98ae.

**Comment by wdoekes** [comment]
See also #52 .
